src/rajAJun20/ExtractItemIdFromInput.java

```diff
@@ -0,0 +1,37 @@
+/* Extract ItemId from below input :
+	 String input = https://www.facebook.com/technocredits/itemId=12345/
+	 String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+	 String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+	 String input = https://www.facebook.com/technocredits/empid=123/
+ */
+
+package rajAJun20;
+
+public class ExtractItemIdFromInput {
+
+	void findItemIdFromInput(String input) {
+		String itemId = "";
+		if (input.contains("itemId")) {
+			String temp = input.substring(input.indexOf("itemId"));
+			for (int index = 0; index < temp.length(); index++) {
+				if (temp.charAt(index) >='0' && temp.charAt(index) <='9') {
+					itemId += temp.charAt(index);
+					if (!(temp.charAt(index + 1) >='0' && temp.charAt(index+1) <='9'))
```

Comment:
This if is not required, you should write else of parent if.
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}